### PR TITLE
Plushie neco

### PR DESCRIPTION
**Pull request by suicidxsaturday**
## About the PR
Added plushie Neco.
Adds a collection of new sounds to Meow , using 2 existing sounds in the game and one new.

**Media**

https://youtu.be/3yjTolORDnA

изображение

- [X] I have added screenshots/videos to this PR showcasing its changes ingame

**Changelog**

:cl:
- add: Added Neco plushie.
- add: Added a new sound "meow" collection.

### CODE REVIEW

**Comment by UbaserB** [comment]
this does not look like itd fit into the game

**Comment by Anzarot121** [comment]
I'm not sure if this fits the game's license. Does anyone know if the Neco Arc design is a registered intellectual property?

**Comment by KaiShibaa** [comment]
This is probably going to cause copyright issues and I agree, it doesn't fit.

File: Resources/Textures/Objects/Fun/toys.rsi/meta.json
Comment (ficcialfaint) [inline Resources/Textures/Objects/Fun/toys.rsi/meta.json:1]: it definitely shouldn't have that big diff

File: Resources/Textures/Objects/Fun/toys.rsi/plushieneco.png
Comment (ficcialfaint) [inline Resources/Textures/Objects/Fun/toys.rsi/plushieneco.png:1]: IMO they look more like a mob than a plushie

File: Resources/Audio/Items/Toys/meow3.ogg
Comment (ficcialfaint) [inline Resources/Audio/Items/Toys/meow3.ogg:1]: `Attributions.yml` need to also include this file

File: Resources/Audio/Items/attributions.yml [hunk 1]
```diff
@@ -52,3 +52,8 @@
   license: "CC-BY-SA-3.0"
   copyright: "Taken from /tg/station, edited by Flareguy for Space Station 14"
   source: "https://github.com/tgstation/tgstation/blob/6665eec76c98a4f3f89bebcd10b34b47dcc0b8ae/sound/weapons/armbomb.ogg"
+
+- files: ["Toys/meow2.ogg"]
+  license: "CC0-1.0"
+  copyright: "Created by suicdxsaturday, shortened and converted to OGG and mono"
+  source: "https://freesound.org/people/suicdxsaturday/sounds/698633/"
```
Comment (ficcialfaint) [inline Resources/Audio/Items/attributions.yml:59]: Was it recorded by you (your cat) or did you just take it from somewhere and uploaded on freesound?

**Review by ficcialfaint — CHANGES_REQUESTED** [review]
Have you ensured using them in the game is not against their license? 
firefox_YIJZUHL6mb

File: Resources/Audio/Items/Toys/meow3.ogg
Comment (ficcialfaint) [reply Resources/Audio/Items/Toys/meow3.ogg:1]: do not

File: Resources/Audio/Items/attributions.yml (hunk 1 above)
Comment (suicidxsaturday) [reply Resources/Audio/Items/attributions.yml:59]: as written in the file, only it was free and im downloaded and "shortened and converted to OGG and mono"

Comment (ficcialfaint) [reply Resources/Audio/Items/attributions.yml:59]: It isn't said in this file that is was free and you just made it look the sound is yours.

**Comment by KaiShibaa** [comment]
Even if approved via licensing. I still think it doesn't fit.

File: Resources/Audio/Items/attributions.yml (hunk 1 above)
Comment (suicidxsaturday) [reply Resources/Audio/Items/attributions.yml:59]: That is, what kind of license do I need to specify all the same? I actually changed the sound...

File: Resources/Audio/Items/Toys/meow3.ogg
Comment (suicidxsaturday) [reply Resources/Audio/Items/Toys/meow3.ogg:1]: So if the sound is already in the game, why should I prescribe to it. It also existed as a "cat_meow"

File: Resources/Audio/Items/attributions.yml (hunk 1 above)
Comment (ficcialfaint) [reply Resources/Audio/Items/attributions.yml:59]: Look at other sounds and how they license it, nobody wipes out the OG author of the sound even if you modified it. And answer the most important question about the sprites copyright/license

File: Resources/Audio/Items/Toys/meow3.ogg
Comment (ficcialfaint) [reply Resources/Audio/Items/Toys/meow3.ogg:1]: Then why do you copy it instead of just using it??? Do you not like its name that much?

Comment (suicidxsaturday) [reply Resources/Audio/Items/Toys/meow3.ogg:1]: I made all 3 sounds together, as for me it's more convenient, but i can change

File: Resources/Audio/Items/attributions.yml (hunk 1 above)
Comment (suicidxsaturday) [reply Resources/Audio/Items/attributions.yml:59]: The sprite is freely available on the Internet. But still i tried to redraw it, in NecoBlack i tried to make more changes. But if you think that it does not fit into the game, then this is your opinion.

Comment (suicidxsaturday) [reply Resources/Audio/Items/attributions.yml:59]: The sprite is freely available on the Internet. But still i tried to redraw it, in NecoBlack i tried to make more changes. But if you think that it does not fit into the game, then this is your opinion.

**Comment by asperger-sind** [comment]
I'd say this needs to be converted to ss14's artstyle before being merged, otherwise this is silly and I think could be a rare maints loot.

**Comment by deathride58** [comment]
Why do these sprites look suspiciously like Neco-Arc's sprites used in MBAA? Like. Right down to having the exact same pattern of bangs (This is an eyebrow-raiser since Neco-Arc's hair is never consistent throughout Type-Moon's works)

They don't appear to be direct rips (Neco-Arc in MBAA is around 40x40, whilst these Neco-Arcs are 30x30), but were these sprites traced?

File: Resources/Audio/Items/attributions.yml (hunk 1 above)
Comment (suicidxsaturday) [reply Resources/Audio/Items/attributions.yml:59]: "Also, there is no problem with the use of newly drawn illustrations based on images of TYPE-MOON works"

The sprite is not from the original "painting", Yes, it is based on the works of TYPE-MOON, I don't really understand, is there a problem?

**Comment by Equivocateur** [comment]
slap an april fools flair and let it cook

**Pull request closed without merging**